src/renderer/services/transaction/transactionService.ts

```diff
@@ -440,6 +442,12 @@ export const useTransaction = (): ITransactionService => {
       });
   };
 
+  const verifySignature = (payload: string | Uint8Array, signature: HexString, accountId: AccountId): Boolean => {
+    const result: VerifyResult = signatureVerify(payload, signature, accountId);
+
+    return result.isValid;
```

Comment:
Not sure about `result` variable, can use inline function call
```suggestion
    return signatureVerify(payload, signature, accountId).isValid;
```